Source/commands/globals.ts

```diff
@@ -0,0 +1,7 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Dolittle. All rights reserved.
+*  Licensed under the MIT License. See LICENSE in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+import { CommandCoordinator } from './internal'
+
+export let commandCoordinator = new CommandCoordinator();
```

Comment:
We should not provide globals of anything in our library - that would be up to the consumer of the library to decide on. We want to advocate dependency inversion instead.

Reply:
This is kind of the pattern for “dependency injection” that I’ve used for my other projects. I think it makes sense for now, especially with those classes which might act as singletons ( for example the coordinators)

Reply:
We should not do this for frameworks, it is the consumer that decides the lifecycle. For the other scenarios we’ve accepted this, it makes more sense - but this is a generic framework piece and we don’t want to decide the lifecycle here. They are not global singletons decided by the library at this time.

Reply:
Makes sense, I’ll remove it